src/run/run.c

```diff
@@ -845,6 +849,8 @@ static int parse_argv_sudo_mode(int argc, char *argv[]) {
                 { "group",               required_argument, NULL, 'g'                     },
                 { "nice",                required_argument, NULL, ARG_NICE                },
                 { "chdir",               required_argument, NULL, 'D'                     },
+                { "via-shell",           no_argument,       NULL, ARG_VIA_SHELL           },
+                { "login",               no_argument,       NULL, 'i'                     }, /* compat with sudo, --via-shell + --chdir='~' */
```

Comment:
`"login"` is neither in the help message nor man page.

Reply:
Only supported for compat with sudo, as the comment says. Its weirdly named hence not advertised in man page and help text

Reply:
Ah, I did not read the commit message. Thanks.